lib/Client.js

```diff
@@ -179,10 +178,20 @@ class Client extends EventEmitter {
         this.reconnectAttempts = 0;
     }
 
+    get token() {
+        return this._token;
+    }
+
     get uptime() {
         return this.startTime ? Date.now() - this.startTime : 0;
     }
 
+    _validateToken() {
```

Comment:
Probably should be renamed. To me, `_validateToken` implies it's checking for proper format without making any changes. Perhaps `_formatToken`?